lib/configuration.rb

```diff
@@ -5,9 +5,17 @@ class Configuration
   def initialize
     logger = LogStashLogger.new(type: :stdout)
 
-    # no aws?  no problem.  Assume that this is development machine, and support builds and api
-    self.service_names = ['jockey-api-development', 'jockey-build-development']
+    # no aws?  no problem. Pass in service names and system services via ENV
+    self.service_names = []
+    if ENV['SERVICE_NAMES']
+      self.service_names = ENV['SERVICE_NAMES'].split(',')
+    end
+
     self.system_services = []
+    if ENV['SYSTEM_SERVICES']
+      self.system_services = ENV['SYSTEM_SERVICES'].split(',')
+    end
```

Comment:
In production, they are passed in via [ec2 user data](https://github.com/bellycard/docker_consul_update/blob/master/lib/configuration.rb#L13-L21). Previously in dev they were hard-coded in code. I'm changing it to be based in by ENV variables.